repo/blob/s3/s3_storage.go

```diff
@@ -181,6 +181,7 @@ func (s *s3Storage) putBlob(ctx context.Context, b blob.ID, data blob.Bytes, opt
 		StorageClass:    storageClass,
 		RetainUntilDate: retainUntilDate,
 		Mode:            retentionMode,
+		PartSize:		 s.Options.UploadPartSize,
```

Comment:
this should be auto-formatted to align with the line above.

Reply:
I ran `go fmt` on that file and did a force-push.